packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -198,23 +198,12 @@ class ScaffoldMessengerState extends State<ScaffoldMessenger> with TickerProvide
       Queue<ScaffoldFeatureController<SnackBar, SnackBarClosedReason>>();
   AnimationController? _snackBarController;
   Timer? _snackBarTimer;
-  bool? _accessibleNavigation;
+  late bool? _accessibleNavigation;
```

Comment:
why do we need `late`? or do you forgot to remove the nullable?

Reply:
Or change it to `bool _accessibleNavigation = false;`?